cypress/globals.d.ts

```diff
@@ -19,59 +19,61 @@ export type CreateUserParams = {
   }
 }
 
+declare global {
```

Comment:
I added this because I was getting a bunch of intellisense errors about our custom commands not existing